ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -330,7 +325,7 @@ class WalletHomeScreen extends React.PureComponent<Props, State> {
     if (
       this.props.allActiveBonus.length === 0 ||
       this.props.allActiveBonus.every(ab => isStrictSome(ab)) ||
-      isReady(this.props.bpdActiveBonus)
+      pot.isLoading(this.props.bpdActiveBonus)
```

Comment:
Why in this case you didn't use `pot.isSome`?

Reply:
I think it is a typo done during porting 😄 wrong for sure! Changed in f297755